protobuf/README.md

```diff
@@ -0,0 +1,85 @@
+# Protocol Buffers + Docker
+
+A lightweight protoc Docker image, published as otel/build-protobuf to Docker Hub,
+with all dependencies built-in, to generate code in multiple languages.
+
+Forked from https://github.com/jaegertracing/docker-protobuf.
+
+## What's included in the image
+
+- <https://github.com/ckaznocha/protoc-gen-lint>
+- <https://github.com/gogo/protobuf>
+- <https://github.com/golang/protobuf>
+- <https://github.com/google/protobuf>
+- <https://github.com/grpc-ecosystem/grpc-gateway>
+- <https://github.com/grpc/grpc>
+- <https://github.com/grpc/grpc-java>
+
+## Supported languages
+
+- C#
+- C++
+- Go
+- Java / JavaNano (Android)
+- JavaScript
+- Objective-C
+- PHP
+- Python
+- Ruby
+
+## Usage
+
+```bash
+docker run --rm -v<some-path>:<some-path> -w<some-path> otel/build-protobuf [OPTION] PROTO_FILES
+```
+
+For help try:
+
+```bash
+docker run --rm otel/build-protobuf --help
+```
+
+### To generate language specific code
```

Comment:
btw, do we keep these in proto files?

https://github.com/open-telemetry/opentelemetry-proto/blob/dfa8c325321d55cd9e27fe6e2c7bf927bb4d9ca3/opentelemetry/proto/collector/logs/v1/logs_service.proto#L24-L27

Reply:
Those are proto options, here I am talking about external proto files dependencies.